app/assets/stylesheets/dashboard.scss

```diff
@@ -0,0 +1,9 @@
+.dashboard_partial{
+  border: 2px solid #003A5B;
```

Comment:
A thing one might want to consider is that Sass lets you DRY up your CSS.  As your CSS codebase grows, things like commonly repeated colors can be put into [variables][1].  You can even abstract things like that border into a [mixin][2] if you're feeling fancy.

[1]:  https://robots.thoughtbot.com/sass-variables
[2]:  http://www.sitepoint.com/sass-basics-the-mixin-directive/